src/stylesheets/partials/_header.scss

```diff
@@ -3,21 +3,20 @@
   // Set the header bar color to official USGS blue
   background-color: $usgs-blue;
   // Put the header bar in a container and control the width of that container so that it will not stretch the whole width of the page
-  .usgs-header-container {
-    @include grid-container($theme-site-max-width);
-  }
+  //.usgs-header-container {
```

Comment:
Can these comentted-out lines be removed?

Reply:
Yes. My bad.